apps/hash-frontend/next.config.js

```diff
@@ -36,6 +36,9 @@ process.env.NEXT_PUBLIC_SENTRY_DSN = process.env.SENTRY_DSN ?? "";
 process.env.NEXT_PUBLIC_SENTRY_REPLAY_SESSION_SAMPLE_RATE =
   process.env.SENTRY_REPLAY_SESSION_SAMPLE_RATE ?? 1;
 
+process.env.NEXT_PUBLIC_NOTIFICATION_POLL_INTERVAL =
+  process.env.NOTIFICATION_POLL_INTERVAL;
```

Comment:
Whilst testing locally, this broke polling for me entirely when the `NOTIFICATION_POLL_INTERVAL` is not set in `.env.local`. It seems that when the value of it is `undefined` here, without a `?? ""` next stringifies undefined to `"undefined"`, which is then passed to `parseInt` resulting in `NaN` instead of the default `10000` polling interval.

```suggestion
process.env.NEXT_PUBLIC_NOTIFICATION_POLL_INTERVAL =
  process.env.NOTIFICATION_POLL_INTERVAL ?? "";
```